initfiles/componentfiles/configxml/buildsetCC.xml.in

```diff
@@ -44,6 +44,11 @@
              processName="DaliServerProcess"
              schema="dali.xsd"/>
    <BuildSet installSet="deploy_map.xml"
+             name="daliplugin"
```

Comment:
I wonder if we're being descriptive enough with the component name/type... if this is the cassandra based WU Server, maybe this should be name should somehow reflect that?